reel-picker/src/main/java/com/innowisegroup/reelpicker/datetime/LocalTime.kt

```diff
@@ -1,10 +1,28 @@
 package com.innowisegroup.reelpicker.datetime
 
+import com.innowisegroup.reelpicker.extension.isWithinMinMaxRange
 import com.innowisegroup.reelpicker.extension.requireNonNull
 import java.io.Serializable
 import java.util.*
 
-class LocalTime(val hour: Int, val minute: Int) : Serializable {
+class LocalTime(hour: Int, minute: Int) : Serializable {
+
+    internal var hour = hour
+        set(value) {
+            validateHour(hour)
+            field = value
+        }
+
+    internal var minute = minute
+        set(value) {
+            validateMinute(minute)
+            field = value
+        }
+
+    init {
+        this.hour = hour
+        this.minute = minute
+    }
 
     @JvmName("getHourKotlin")
     fun getHour() = requireNonNull(hour)
```

Comment:
what the purpose of @JvmName here and method naming "getHourKotlin". 
The same for getMinute function

Try to remove this functions and reference hour via "this.hour" and minute via "this.minute". There should't be conflicts then.

    internal fun withHour(hour: Int): LocalTime =
        if (this.hour == hour) this else create(hour, this.minute)

    internal fun withMinute(minute: Int): LocalTime =
        if (this.minute == minute) this else create(this.hour, minute)